components/ui/atoms/list-card-item/context/cardItemProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import { CardItemTransformed } from "@/types/cardItemType";
+import { createContext, useContext, ReactNode, useMemo } from "react";
+
+interface CardProviderProps {
+  cardData: CardItemTransformed;
+  children: ReactNode;
+}
+
+const CardContext = createContext<CardItemTransformed | undefined>(undefined);
```

Comment:
```suggestion
const CardContext = createContext<CardItemTransformed >(undefined);
```
제가 알기로 TS에서 undefined로 초기값을 넣으면 자동으로 undefined를 추론해서 이렇게 작성해도 되는걸로 알고 있어요 만약에 아니라면 그대로 넣어주세요!